packages/repeater/src/lib/RepeaterFactory.ts

```diff
@@ -38,4 +37,18 @@ export class RepeaterFactory {
       configuration: this.configuration
     });
   }
+
+  private registerRunnerOptions(
```

Comment:
method does much more than just registering `RequestRunnerOptions`: it creates container with not only options in it. consider rework and better naming